src/typed-method-types/typed-method-tests.ts

```diff
@@ -0,0 +1,12 @@
+import { assertEquals } from "https://deno.land/std@0.99.0/testing/asserts.ts";
+import { SlackAPI } from "../mod.ts";
+
+Deno.test("Custom Type Methods are valid functions", () => {
```

Comment:
Maybe we want to generate these tests based on the contents of `methodsWithCustomTypes` from `src/typed-method-types/mod.ts`? Rather than hard-coding the list in both files?